README.md

```diff
@@ -450,7 +488,7 @@ remote IEx feature.
 
 ## CA Certificates
 
-If you are connecting to the publicly available https://nerves-hub.org instance, then
+If you are connecting to the publicly available <https://nerves-hub.org> instance, then
```

Comment:
This paragraph seems out of date. I'm not sure if this edit is intentional or not, though.

Reply:
Whoops, i didn't do that. i guess my editor did that..

Reply:
I removed this change, but i just read the entire paragraph and now realize what you mean by it being out of date. I'll follow up after this PR with an update to this paragraph